### ISSUE TITLE
Proposal: Responsive screen-reader utility classes

### ISSUE THREAD

**Issue by Josh68** [CMSgov/design-system#1539]
## What
> Propose the DS adds [responsive variations](https://design.cms.gov/utilities/display-visibility/#utilities.display-visibility.responsive) of the screen-reader utility class
> This may add a little bloat to the CSS output - it cannot use a `placeholder` or `@extends` syntax, but can work with a `mixin` and `@include`
> [The declaration block](https://github.com/CMSgov/design-system/blob/dc1f98be2afb6cbbde777acd22f4ee35bd3c9f37/packages/design-system/src/styles/utilities/_display-visibility.scss#L32) could be defined as a mixin and used inside [this breakpoint loop](https://github.com/CMSgov/design-system/blob/dc1f98be2afb6cbbde777acd22f4ee35bd3c9f37/packages/design-system/src/styles/utilities/_display-visibility.scss#L53)

## Why
> Explain why you think this should be added to the Design System.
> - In MCT, we have scenarios where we want to hide content, usually at lower breakpoints, but keep exposed to screen readers
